golang/logging/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  logging:
+    build:
+      context: ./images/logging
+    command:  go run .
+    ports: 
+      - '40000:40000'
+    volumes:
+      - ./src:/go/src/logging
+  db:
+    image: mysql
+    command: mysqld --max_allowed_packet=256M --max_connections=1024 --default-authentication-plugin=mysql_native_password
+    container_name: logging_db
+    environment:
+      MYSQL_DATABASE: logging
+      MYSQL_ALLOW_EMPTY_PASSWORD: 'yes'
+    ports:
+    - '3307:3306'
+  lb:
+    build:
+      context: ./images/nginx
+    command: /service/entrypoint.sh
```

Comment:
This is going to need it's own entrypoint to wait for the logging service, otherwise it will try to wait for day_trader_web